src/tools/UpdatePackageDependencies.ts

```diff
@@ -155,7 +137,55 @@ export async function updatePackageDependencies() : Promise<void> {
     fs.writeFileSync('package.json', content);
 }
 
-function getLowercaseFileNameFromUrl(url : string) : string {
+
+function replaceVersion(value: string, newVersion: string): string {
+    if (!value) {
+        return value; //if the value is null or undefined return the same one
+    }
+
+    let regex: RegExp = dottedVersionRegExp;
```

Comment:
This code is working really hard to prevent a version number from changing from dashed to dotted. How important is it to enforce that?

Reply:
This part of the code isn't new. And it is extremely important - if it didn't do this, the resulting URL would be wrong.

Reply:
Ok, cool.